frontend/packages/knative-plugin/integration-tests/utils/install-serverless-operator.ts

```diff
@@ -0,0 +1,107 @@
+import { execSync } from 'child_process';
+import { protractor, by, browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName, retry } from '../../../../integration-tests/protractor.conf';
+import {
+  installServerlessOperator,
+  firstKebabMenu,
+  machineSetsInput,
+  submitCount,
+  servicesValidation,
+  revisionsValidation,
+  machineCount,
+  knativeServingNS,
+  pageSidebar,
+} from '../views/serverless.view';
+
+import {
+  switchPerspective,
+  Perspective,
+  sideHeader,
+} from '../../../dev-console/integration-tests/views/dev-perspective.view';
+import * as sidenavView from '../../../../integration-tests/views/sidenav.view';
+import * as crudView from '../../../../integration-tests/views/crud.view';
+import * as catalogView from '../../../../integration-tests/views/catalog.view';
+import * as catalogPageView from '../../../../integration-tests/views/catalog-page.view';
+import * as operatorView from '../../../operator-lifecycle-manager/integration-tests/views/operator.view';
+import * as operatorHubView from '../../../operator-lifecycle-manager/integration-tests/views/operator-hub.view';
+
+export const installKnOperator = async () => {
```

Comment:
Make this function generic so that it can be used to install other operator as well